src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -12332,8 +12332,8 @@ namespace OutputReportTabular {
                   "1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36\n");
             // Put the decay curve into the EIO file
             for (int iZone = 1; iZone <= NumOfZones; ++iZone) {
-                ZoneData &zd(Zone(iZone));
-                for (int kSurf = zd.SurfaceFirst; kSurf <= zd.SurfaceLast; ++kSurf) {
```

Comment:
All changes in OutputReportTabular and OutputReports are to maintain surface order in output reports.